CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
```

Comment:
Does this need the newest version in it?

Reply:
I'll add "Added support for Swift Package Manager" when I tag 0.5.0